### PR TITLE
Developed the new scan webpage

### LINKED ISSUE #13: Develop HTML + CSS Flask templates for adding a new scan

### PR DESCRIPTION
**Pull request by maanas-talwar**
Closes #13

Screenshots for reference
=======
## Mobile View:

@kmehant @1209973 @Ammoniya I would request you to please review this pull request and share your feedback.
Thanks

### CODE REVIEW

File: Dashboard/templates/newScan.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            var numFiles = document.getElementById('customFile').files.length;
+            if(numFiles > 0) {
+                document.getElementById('upload').disabled = false;
+            } else {
+                document.getElementById('upload').disabled = true;
+            }
+            var msg;
+            if (numFiles == 0) {
+                msg = "Choose File";
+            } else if (numFiles == 1) {
```
Comment (kmehant) [inline Dashboard/templates/newScan.html:55]: @maanas-talwar Why do you need this else if block? Can we remove this?

**Review by kmehant — CHANGES_REQUESTED** [review]

File: Dashboard/templates/newScan.html (hunk 1 above)
Comment (maanas-talwar) [reply Dashboard/templates/newScan.html:55]: It is just for the sake of "1 file uploaded" and not "1 file(s) uploaded"

Comment (maanas-talwar) [reply Dashboard/templates/newScan.html:55]: If the other way suits better I can remove the else if block

File: Dashboard/templates/newScan.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                document.getElementById('upload').disabled = false;
+            } else {
+                document.getElementById('upload').disabled = true;
+            }
+            var msg;
+            if (numFiles == 0) {
+                msg = "Choose File";
+            } else {
+                msg = numFiles + ' file(s) uploaded';
+            }
```
Comment (kmehant) [inline Dashboard/templates/newScan.html:57]: @maanas-talwar  Why two different if-else blocks, can't we combine them into one?

**Review by kmehant — CHANGES_REQUESTED** [review]

File: Dashboard/templates/newScan.html (hunk 2 above)
Comment (maanas-talwar) [reply Dashboard/templates/newScan.html]: Sure

**Review by kmehant — APPROVED** [review]
LGTM!

**Pull request merged**